DataFormats/SiStripCluster/src/classes_def.xml

```diff
@@ -23,4 +23,21 @@
  <class name="edm::Wrapper<edmNew::DetSetVector<edm::Ref<edmNew::DetSetVector<SiStripCluster>,SiStripCluster,edmNew::DetSetVector<SiStripCluster>::FindForDetSetVector> > >" />
 
 
+<class name="SiStripApproximateCluster"/>
```

Comment:
minor, but does the indendation here have significance? perhaps the following would reduce confusion
```suggestion
 <class name="SiStripApproximateCluster"/>
```

Reply:
a class version (3 for the first) is needed

Reply:
added version and fixed indentation

Reply:
added